InnerEye/ML/configs/histo_configs/classification/BaseMIL.py

```diff
@@ -47,6 +49,10 @@ class BaseMIL(LightningContainer):
                                                   "dataset upfront and save it to disk.")
     # local_dataset (used as data module root_path) is declared in DatasetParams superclass
 
+    # slide dataset parameters:
```

Comment:
probably worth to mention these parameters are only used for vizualizations

Reply:
these have been removed from here following the suggestion below